lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -6,12 +6,13 @@
 
 from __future__ import annotations
 
+import collections.abc as c
 import os
 import pathlib
 import typing as t
 
 from collections import namedtuple
-from collections.abc import MutableSequence, MutableMapping
+from collections.abc import Hashable, MutableSequence
```

Comment:
Instead of importing these, just refer to them below as `c.Hashable` and `c.MutableSequence`.

Reply:
Done locally. Marking this as resolved.